core/testsuite/src/test/java/com/blazebit/persistence/testsuite/DateExtractTest.java

```diff
@@ -94,12 +89,7 @@ public DateExtractTest(TimeZone producerTimeZone, TimeZone clientTimeZone) {
 
     @Override
     protected boolean recreateDataSource() {
-        // Some drivers have timezone information bound to the connection
-        // So we have to recreate the data source to get the newly configured time zones for connections
-        boolean recreate = previousProducerTimeZone != null && previousClientTimeZone != null && (!producerTimeZone.equals(previousProducerTimeZone) || !clientTimeZone.equals(previousClientTimeZone));
-        previousProducerTimeZone = producerTimeZone;
-        previousClientTimeZone = clientTimeZone;
-        return recreate;
+        return true;
```

Comment:
Shouldn't this be reverted?

Reply:
No, the date extract tests are not reliable without this. Tests started failing after adding the epoch_seconds alias for epoch. Even though they use the same function implementation and test logic, the tests fail if both tests for epoch snd epich seconds are ran without this change.